src/main/resources/interface-specification.yml

```diff
@@ -806,6 +806,9 @@ components:
         id:
           type: string
           format: uuid
+        riskAnalysisId:
```

Comment:
should this be added also in the seed?

Reply:
Yes, this should be passed from process.